apps/team/src/pages/api/team/all.ts

```diff
@@ -1,16 +1,11 @@
 import csv from "csvtojson";
 import pLimit from "p-limit";
-import { imageCache } from "lib/models";
-import { hash } from "lib/hash";
-import { authorizedEndpoint, Fetcher } from "lib/auth";
-import { urlFromReq } from "lib/url";
+import { imageCache } from "@artsy-studio/models";
+import { authorizedEndpoint, Fetcher } from "@artsy-studio/auth";
+import { urlFromReq, hash, capitalize } from "@artsy-studio/utils";
```

Comment:
`capitalize` something that lodash can handle?

Reply:
Totally is. Just didn't think it was worth adding the dependency.

Reply:
Added lodash-es, deleted this.